maven/bnd-indexer-maven-plugin/src/main/java/aQute/bnd/maven/indexer/plugin/LocalIndexerMojo.java

```diff
@@ -26,14 +26,23 @@
 public class LocalIndexerMojo extends AbstractMojo {
 	private static final Logger	logger		= LoggerFactory.getLogger(LocalIndexerMojo.class);
 
+	private static final String ABS_URL_IDENTIFIER	= "///";
```

Comment:
Instead of some magic file name, it would be better to specify a configuration setting to use absolute Files and then when set you can process `toIndex` to convert the Files to absolute/canonical before passing to `SimpleIndexer`

Reply:
Would be an option. Can you check my latest commit and comment if it should be changed.